header/scss/_marketing-promo.scss

```diff
@@ -23,7 +23,7 @@
 		vertical-align: middle;
 		text-align: right;
 		padding-right: 120px;
-		background: url('https://next-geebee.ft.com/image/v1/images/raw/https%3A%2F%2Fnext-geebee.ft.com%2Fassets%2Fproduct%2Fpack-shot-next-2.png?source=next&height=100') right top no-repeat;
+		background: url('https://www.ft.com/__origami/service/image/v2/images/raw/https%3A%2F%2Fnext-geebee.ft.com%2Fassets%2Fproduct%2Fpack-shot-next-2.png?source=next&height=100') right top no-repeat;
```

Comment:
Because it allows us to automatically bypass preflight, and indicates an internal tool. We changed to this after some discussion with @phamann and @richard-still-ft.